scene/resources/3d/primitive_meshes.cpp

```diff
@@ -249,6 +249,8 @@ void PrimitiveMesh::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_uv2_padding", "uv2_padding"), &PrimitiveMesh::set_uv2_padding);
 	ClassDB::bind_method(D_METHOD("get_uv2_padding"), &PrimitiveMesh::get_uv2_padding);
 
+	ClassDB::bind_method(D_METHOD("_request_update"), &PrimitiveMesh::_request_update);
```

Comment:
If it's exposed to be user accessible, it probably should be `request_update` without underline. Also, it's already being exposed in the subclass `TextMesh` (should be removed, or it will cause conflict), but it was exposed only to allow `call_deffered` (this case can be replaced with `callable_mp`).

Reply:
If we un-expose `_request_update()` from TextMesh will it break binary compatibility for extensions?

Reply:
It doesn't seem to be exposed in https://raw.githubusercontent.com/godotengine/godot-cpp/master/gdextension/extension_api.json

I believe we explicitly skip non virtual methods prefixed with underscore.

Reply:
> but it was exposed only to allow `call_deffered` (this case can be replaced with `callable_mp`).

It looks like `callable_mp`  can not use parent class's protected method, should I make `_request_update` public?

Reply:
> It looks like callable_mp can not use parent class's protected method, should I make _request_update public?

Probably, it's also make sense to make it public, if it's exposed. Removing `_` should not break compatibility, since `_` methods are considered private and not fully exposed to GDExtension.

Reply:
For consistency, both the exposed method and the now public C++ one should be renamed `request_update` (no underscore).